samples/Commerce/Commerce.UI/Views/FilterPage.xaml

```diff
@@ -110,7 +110,7 @@
 				<Button Content="Apply"
 						x:Uid="Body_Apply"
 						utu:AutoLayout.PrimaryAlignment="Stretch"
-						Style="{StaticResource MaterialContainedButtonStyle}"
+						Style="{StaticResource ContainedButtonStyle}"
```

Comment:
Contained is now either Elevated, Filled, or Filled Tonal

https://m3.material.io/components/buttons/overview